innovation.game.php

```diff
@@ -8386,6 +8392,54 @@ function stPlayerInvolvedTurn() {
             case "182N1":
                 $step_max = 1;
                 break;
+
+            // id 185, Artifacts age 8: Parnell Pitch Drop
+            case "185N1":
+                // "Draw and meld a card of value one higher than the highest top card on your board."
+                $card = self::executeDraw($player_id, self::getMaxAgeOnBoardTopCards($player_id) + 1, 'board');
+                $count_clocks = 0;
+                if($card['spot_1'] == 6){
+                    $count_clocks++;
+                }
+                if($card['spot_2'] == 6){
+                    $count_clocks++;
+                }
+                if($card['spot_3'] == 6){
+                    $count_clocks++;
+                }
+                if($card['spot_4'] == 6){
+                    $count_clocks++;
+                }                
+                if ($count_clocks == 3) {
```

Comment:
Can you refactor this into a countResources($card, $icon) function? You could put it right near the existing hasRessource() function.

Reply:
I didn't think this would come up again, but I can do that.  There are no cards in echoes that do this that I know of.